packages/ilios-common/addon/components/dashboard/courses-calendar-filter.js

```diff
@@ -95,9 +95,11 @@ export default class DashboardCoursesCalendarFilterComponent extends Component {
 
   @action
   scrollToLastYear(element, [year]) {
```

Comment:
The method `scrollToLastYear` sounds like "go to 2023, if it is 2024". Should this not be `scrollToLatestYear` or `scrollToMostRecentYear` or `scrollToCurrentYear`?